tests/tests.go

```diff
@@ -0,0 +1,3 @@
+package tests
```

Comment:
Tests is a bad name for this package.
It is not clear to me what it is supposed to be.

Reply:
it's going to be the package with all things that can be shared among tests

Reply:
You might wanna chose a more expressive name then.